src/websocket_service/handlers/push_delete.rs

```diff
@@ -26,90 +28,61 @@ pub async fn handle(
     let database = &state.database;
     let subscription_id = params.id;
 
-    match database
+    let Ok(Some(LookupEntry {
+        project_id, 
+        account,
+        ..
+    }))= database
         .collection::<LookupEntry>("lookup_table")
-        .find_one_and_delete(doc! {"_id": &topic.clone().to_string() }, None)
+        .find_one_and_delete(doc! {"_id": &topic.to_string() }, None)
         .await
-    {
-        Ok(Some(LookupEntry {
-            project_id,
-            account,
-            ..
-        })) => {
-            match database
-                .collection::<ClientData>(&project_id)
+        else {
+            return Err(Error::NoProjectDataForTopic(topic.to_string()))
+        };
+
+    let Ok(Some(acc)) = database
+                 .collection::<ClientData>(&project_id)
```

Comment:
`rustfmt` can't handle this new syntax yet, try to format it manually until it's fixed